packages/ui/atomic/health-check/e2e/test.spec.ts

```diff
@@ -0,0 +1,174 @@
+import stripAnsi from 'strip-ansi';
+import {ChildProcess} from 'node:child_process';
+import {join} from 'node:path';
+import {mkdirSync, readFileSync, writeFileSync} from 'node:fs';
+import {npmSync} from '@coveo/do-npm';
+import {startVerdaccio} from '@coveo/verdaccio-starter';
+import {hashElement} from 'folder-hash';
+import {DirResult, dirSync} from 'tmp';
+import treeKill from 'tree-kill';
+
+const PACKAGE_NAME = '@coveo/create-atomic-component';
+const PACKAGE_NAME_HEALTH = '@coveo/atomic-component-health-check';
+
+describe(PACKAGE_NAME, () => {
+  let verdaccioProcess: ChildProcess;
+  let tempDirectory: DirResult;
+  let testDirectory: string;
+  let npmCache: string;
+  let verdaccioUrl: string;
+
+  beforeAll(async () => {
+    ({verdaccioUrl, verdaccioProcess} = await startVerdaccio([
+      PACKAGE_NAME_HEALTH,
+      PACKAGE_NAME,
+    ]));
+    tempDirectory = dirSync({unsafeCleanup: true, keep: true});
+    npmCache = join(tempDirectory.name, 'npm-cache');
+    mkdirSync(npmCache);
+  });
+
+  afterAll(async () => {
+    verdaccioProcess.pid && (await treeKill(verdaccioProcess.pid));
+    console.log(tempDirectory.name);
+    tempDirectory.removeCallback();
+  });
+
+  describe.each([
+    {
+      describeName: 'when called without any args',
+      args: [],
+      testDirname: 'no-args',
+      packageName: '@coveo/sample-component',
+    },
+    {
+      describeName: 'when called in an existing project',
+      args: ['oh-wow-another-component-can-you-believe-it'],
+      testDirname: 'no-args',
+      packageName: 'oh-wow-another-component-can-you-believe-it',
+    },
+    {
+      describeName: 'when called with a valid component name',
+      args: ['valid-component-name'],
+      testDirname: 'valid-arg',
+      packageName: 'valid-component-name',
+    },
+    {
+      describeName: 'when called with a component name without hyphen',
+      args: ['nohypen'],
+      testDirname: 'invalid-arg',
+      packageName: 'atomic-nohypen',
+    },
+  ])('$describeName', ({args, testDirname, packageName}) => {
```

Comment:
I think you do not need to test all the case here for the `health-check` package, it is  redundant.

Reply:
Sorry, I forgot to move the PR to draft while I was refactoring E2E tests.
I was actually removing redundant tests :P